crates/sui-config/src/genesis.rs

```diff
@@ -768,9 +768,6 @@ impl Builder {
             CertifiedCheckpointSummary::new(checkpoint, signatures, &committee).unwrap()
         };
 
-        // Ensure we have signatures from all validators
```

Comment:
do we need to be signed by at least 2f+1 (weighted?)

Reply:
Yes we still have a requirement that 2f+1 stake weighted sign it.